### PR TITLE
[ps.js - zh-Hans] missing translations

### PR DESCRIPTION
**Pull request by asukaminato0721**

### CODE REVIEW

**Review by unicar9 — APPROVED** [review]
Looks good

**Review by sproutleaf — APPROVED** [review]

**Comment by sproutleaf** [comment]
Both @unicar9 and I approved. Tagging @limzykenneth @Qianqianye for merging, thank you!

File: src/content/reference/zh-Hans/p5/createVideo.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      video.volume(0);
+    }
+    </code>
+    </div>
+class: p5
+params:
+  - name: src
+    description: |
+      <p>视频文件的路径，或者为了支持不同浏览器的多个路径数组。</p>
+    type: '字符串|String[]'
```
Comment (sproutleaf) [inline src/content/reference/zh-Hans/p5/createVideo.mdx:88]: String[]->字符串[] 谢谢！

**Review by sproutleaf — CHANGES_REQUESTED** [review]

**Review by sproutleaf — APPROVED** [review]
Approve again after a small error was found!

**Pull request merged**